web/modules/custom/collection/config/install/core.entity_view_display.rdf_entity.collection.default.yml

```diff
@@ -104,35 +102,48 @@ content:
     third_party_settings: {  }
     type: boolean
   field_ar_modification_date:
-    weight: 6
+    weight: 7
     label: above
     settings:
       format_type: medium
       timezone_override: ''
     third_party_settings: {  }
     type: datetime_default
   field_ar_owner:
-    weight: 8
+    weight: 9
     label: above
     settings:
       link: true
     third_party_settings: {  }
     type: entity_reference_label
+  field_ar_shared_content:
+    type: entity_reference_label
+    weight: 2
+    label: above
+    settings:
+      link: true
+    third_party_settings: {  }
```

Comment:
I don't think it is intentional to make this field show up. I'm not even sure what it is used for.